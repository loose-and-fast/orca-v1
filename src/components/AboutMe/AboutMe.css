.AboutMe {
	text-align: center;
	color: white;
	height: 100%;
	background-color: #2d2c2c;
	padding-top: 90px;
}

.AboutMe h1 {
	font-family: 'PT Sans', sans-serif;
	margin: 0px;
}

.AboutMe h6 {
	color: #887409;
	text-align: center;
}

.AboutMe .ipsum-lorem {
	text-align: center;
	padding: 40px 400px;
	color: gray;
}