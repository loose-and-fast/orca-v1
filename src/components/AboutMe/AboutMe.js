import React from "react";
import PropTypes from "prop-types";
import "./AboutMe.css";

const AboutMe = () => (
<div className="AboutMe">
	<h6>SOMETHING</h6>
	<h1>ABOUT ME</h1>
	<div className="ipsum-lorem">
		Lorem ipsum officia aliqua commodo elit labore
		excepteur voluptate in culpa non anim minim quis.
		Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem
		Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis.
		Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.
	</div>
</div>);

AboutMe.propTypes = {};

export default AboutMe;
