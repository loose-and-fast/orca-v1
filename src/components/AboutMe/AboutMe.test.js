import React from "react";
import AboutMe from "./AboutMe";

test("AboutMe renders", () => {
  const props = {};
  const wrapper = shallow(<AboutMe {...props} />);
  expect(wrapper).toMatchSnapshot();
});
