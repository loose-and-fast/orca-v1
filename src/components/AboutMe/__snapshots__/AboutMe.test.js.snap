// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutMe renders 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <AboutMe />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h6>
          SOMETHING
</h6>,
        <h1>
          ABOUT ME
</h1>,
        <div
          className="ipsum-lorem"
>
          Lorem ipsum officia aliqua commodo elit labore excepteur voluptate in culpa non anim minim quis. Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis. Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.
</div>,
      ],
      "className": "AboutMe",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "SOMETHING",
        },
        "ref": null,
        "rendered": "SOMETHING",
        "type": "h6",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "ABOUT ME",
        },
        "ref": null,
        "rendered": "ABOUT ME",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem ipsum officia aliqua commodo elit labore excepteur voluptate in culpa non anim minim quis. Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis. Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.",
          "className": "ipsum-lorem",
        },
        "ref": null,
        "rendered": "Lorem ipsum officia aliqua commodo elit labore excepteur voluptate in culpa non anim minim quis. Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis. Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.",
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h6>
            SOMETHING
</h6>,
          <h1>
            ABOUT ME
</h1>,
          <div
            className="ipsum-lorem"
>
            Lorem ipsum officia aliqua commodo elit labore excepteur voluptate in culpa non anim minim quis. Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis. Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.
</div>,
        ],
        "className": "AboutMe",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "SOMETHING",
          },
          "ref": null,
          "rendered": "SOMETHING",
          "type": "h6",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "ABOUT ME",
          },
          "ref": null,
          "rendered": "ABOUT ME",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem ipsum officia aliqua commodo elit labore excepteur voluptate in culpa non anim minim quis. Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis. Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.",
            "className": "ipsum-lorem",
          },
          "ref": null,
          "rendered": "Lorem ipsum officia aliqua commodo elit labore excepteur voluptate in culpa non anim minim quis. Enim ut in pariatur non ut commodo occaecat fugiat ut nisi adipisicing nostrud nisi eu.lorem Nostrud nisi ex magna est sunt proident tempor voluptate cupidatat dolore ut anim est irure id incididunt quis. Consectetur duis deserunt laborum id do proident adipisicing aute voluptate excepteur consectetur velit.",
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
